py4j-java/src/main/java/py4j/GatewayServer.java

```diff
@@ -754,7 +783,7 @@ protected void startSocket() throws Py4JNetworkException {
 	 * @param interfacesToImplement
 	 * @return
 	 */
-	public Object getPythonServerEntryPoint(Class[] interfacesToImplement) {
+	public Object getPythonServerEntryPoint(@SuppressWarnings("rawtypes") Class[] interfacesToImplement) {
```

Comment:
does the method parameter annotation works for Java 6 and 7? I thought it was a Java 8 thing?